bluebrain/repo-bluebrain/packages/py-visual-cortex-analysis/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyVisualCortexAnalysis(PythonPackage):
+    """Visual cortex analyses"""
+
+    homepage = 'https://bbpgitlab.epfl.ch/circuits/proj120/visual-cortex-analysis'
+    git      = 'git@bbpgitlab.epfl.ch:circuits/proj120/visual-cortex-analysis.git'
+
+    version('develop', branch='master')
```

Comment:
If this is the only version, Spack will **not** track the git SHA it built with.

IMO it would be better to add a version like
```
    version('0.0.0-2022-05-25', commit='blargh')
```
to have something "reproducible".